### ISSUE TITLE
Test and debug the initial packer builds and images

### ISSUE THREAD

**Issue by dongsupark** [kinvolk/flatcar-packer-qemu#2]
I did an initial work on https://github.com/kinvolk/flatcar-packer-qemu/pull/1 to be able to support libvirt vagrant provider for Flatcar Linux. This PR is basically a rewrite based on existing open source projects, such as https://github.com/dyson/packer-qemu-coreos-container-linux or https://github.com/bfraser/packer-coreos-qemu. The result image for the alpha channel is also available on https://app.vagrantup.com/dongsupark/boxes/flatcar-alpha.

It seems to work for now. Though there are still some things to be done.

- [ ] Find (and/or) fix bugs in the packer build. For example:
  - We need to figure out a better way to define the ssh public key defined in `flatcar-linux.json` or in `ignition.json`. This part looks like a hack.
  - We need to host the images on `vagrantup.com` with our own username `flatcar`, instead of a personal account like `dongsupark`.
  - I needed to set `keep_input_artifact` to `true` in the `post-processors` section, to avoid issues of result images disappearing in the end. This part looks weird, but it doesn't work without the option. I'm not sure if it's an upstream packer bug, or an expected behavior. Or there might be a better way to deal with the issue.
  - I already fixed an issue of boot failures by increasing RAM to 2G, and created [an upstream PR](https://github.com/dyson/packer-qemu-coreos-container-linux/pull/1). We need to follow and watch it in the future.
  - Let's go through other configurations defined in `flatcar-linux.json` or `ignition.json`, and figure out if they are really needed. As most of them are copied from other projects, I'm not sure if every defined config is actually necessary.
- [ ] Test result vagrant images, by running `vagrant up --provider=libvirt` on a vagrant cluster.
- [ ] Support other channels, `beta` and `stable`. This can be done only after beta and stable images would be available on our Flatcar web server.

**Comment by dongsupark** [comment]
Closing in favor of https://github.com/flatcar-linux/flatcar-packer-qemu/issues/2
